tutorials/katacoda/thanos/3-receiver/step3.md

```diff
@@ -0,0 +1,54 @@
+# Configure Prometheus Remote Write
+
+Our problem in the last step was that we have have not yet configured Prometheus to `remote_write` to our `Thanos Receive` instance.
+
+We need to tell `prometheus-batcave` & `prometheus-batcomputer` where to write their data to.
+
+## Update Configuration
+
+The docs for this configuration option can be found [here](https://prometheus.io/docs/prometheus/latest/configuration/configuration/#remote_write).
+
+<pre class="file" data-filename="prometheus-batcave.yaml" data-target="replace">
+global:
+  scrape_interval: 5s
+  external_labels:
+    cluster: batcave
+    replica: 0
+
+scrape_configs:
+  - job_name: 'prometheus'
+    static_configs:
+      - targets: ['0.0.0.0:9090']
```

Comment:
The previous configuration uses address `127.0.0.1:9090`. Let's unify the two to be either `127.0.0.1` or `0.0.0.0`. Though the metrics endpoints are the same, different target strings would be two time series, which is confusing to users.

Reply:
Let's standardise on `127.0.0.1` :+1:

Reply:
These also mean two very different things. 0.0.0.0 means all addresses assigned to the machine where as 127.0.0.1 is just one address on the loopback device. Normally the confusion is in standardizing between `localhost` and `127.0.0.1`.

Reply:
Thanks for the detailed explanation @squat!